app/controllers/characters_controller.rb

```diff
@@ -2,4 +2,14 @@ class CharactersController < ApplicationController
   def index
     @characters = Character.all
   end
+
+  def new
+    @new_character_info = CharacterFacade.create_form
+  end
+
+  def create
+    CharacterFacade.create_new(params["name"], params["lvl"], params["race"], params["class"])
+
+    redirect_to "/"
+  end
 end
```

Comment:
This facade wan't catch incomplete information. Need to add reroutes for missing info.